### PR TITLE
Bad changes

### PR DESCRIPTION
**Pull request by markstory**

### CODE REVIEW

**Comment by markstory** [comment]
The following commits had issues. The pattern (?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

File: somefile.py [hunk 1]
```diff
@@ -42,3 +42,10 @@ def run_command(
 def do_thing():
     print 'thinger'
     print 'thing 2'
+
+def do_thing(*args):
```
Comment (markstory) [inline somefile.py]: E302 expected 2 blank lines, found 1

File: somefile.py [hunk 2]
```diff
@@ -42,3 +42,10 @@ def run_command(
...
+  print 'not 4 spaces'
```
Comment (markstory) [inline somefile.py]: E111 indentation is not a multiple of four

File: somefile.py [hunk 3]
```diff
@@ -42,3 +42,10 @@ def run_command(
...
+
+def doThing(arg) :
```
Comment (markstory) [inline somefile.py]: E302 expected 2 blank lines, found 1

Comment (markstory) [inline somefile.py]: E203 whitespace before ':'

File: somefile.py [hunk 4]
```diff
@@ -42,3 +42,10 @@ def run_command(
...
+    do_thing(arg[1
+        arg[2]])
```
Comment (markstory) [inline somefile.py]: E128 continuation line under-indented for visual indent

File: bad.php [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<?php
+function bad_braces() {
```
Comment (markstory) [inline bad.php]: Missing function doc comment

Comment (markstory) [inline bad.php]: Function name "bad_braces" is not in camel caps format

Comment (markstory) [inline bad.php]: Function name "bad_braces" is prefixed with a package name but does not begin with a capital letter

Comment (markstory) [inline bad.php]: Opening brace should be on a new line

File: bad.php [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+    echo 'this'.'is'.'hard to read';
```
Comment (markstory) [inline bad.php]: Expected 1 space before ., but 0 found

Comment (markstory) [inline bad.php]: Expected 1 space after ., but 0 found

File: somefile.py (hunk 4 above)
Comment (markstory) [inline somefile.py]: E901 SyntaxError: invalid syntax

**Comment by stickler-ci-dev** [comment]
The following commits had issues. The pattern (?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

File: somefile.py (hunk 1 above)
Comment (stickler-ci-dev) [inline somefile.py]: E302 expected 2 blank lines, found 1

File: somefile.py (hunk 2 above)
Comment (stickler-ci-dev) [inline somefile.py]: E111 indentation is not a multiple of four

File: somefile.py (hunk 3 above)
Comment (stickler-ci-dev) [inline somefile.py]: E302 expected 2 blank lines, found 1

Comment (stickler-ci-dev) [inline somefile.py]: E203 whitespace before ':'

File: somefile.py (hunk 4 above)
Comment (stickler-ci-dev) [inline somefile.py]: E128 continuation line under-indented for visual indent

File: bad.php (hunk 5 above)
Comment (stickler-ci-dev) [inline bad.php]: Missing function doc comment

Comment (stickler-ci-dev) [inline bad.php]: Function name "bad_braces" is not in camel caps format

Comment (stickler-ci-dev) [inline bad.php]: Function name "bad_braces" is prefixed with a package name but does not begin with a capital letter

Comment (stickler-ci-dev) [inline bad.php]: Opening brace should be on a new line

File: bad.php (hunk 6 above)
Comment (stickler-ci-dev) [inline bad.php]: Expected 1 space before ., but 0 found

Comment (stickler-ci-dev) [inline bad.php]: Expected 1 space after ., but 0 found

File: somefile.py (hunk 4 above)
Comment (stickler-ci-dev) [inline somefile.py]: E901 SyntaxError: invalid syntax

File: somefile.py (hunk 1 above)
Comment (stickler-ci-dev) [inline somefile.py]: E302 expected 2 blank lines, found 1

File: somefile.py (hunk 2 above)
Comment (stickler-ci-dev) [inline somefile.py]: E111 indentation is not a multiple of four

**Comment by stickler-ci-dev** [comment]
The following commits had issues. The pattern (?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

File: somefile.py (hunk 3 above)
Comment (stickler-ci-dev) [inline somefile.py]: E302 expected 2 blank lines, found 1

Comment (stickler-ci-dev) [inline somefile.py]: E203 whitespace before ':'

File: somefile.py (hunk 4 above)
Comment (stickler-ci-dev) [inline somefile.py]: E128 continuation line under-indented for visual indent

File: bad.php (hunk 5 above)
Comment (stickler-ci-dev) [inline bad.php]: Missing function doc comment

Comment (stickler-ci-dev) [inline bad.php]: Function name "bad_braces" is not in camel caps format

Comment (stickler-ci-dev) [inline bad.php]: Function name "bad_braces" is prefixed with a package name but does not begin with a capital letter

Comment (stickler-ci-dev) [inline bad.php]: Opening brace should be on a new line

File: bad.php (hunk 6 above)
Comment (stickler-ci-dev) [inline bad.php]: Expected 1 space before ., but 0 found

Comment (stickler-ci-dev) [inline bad.php]: Expected 1 space after ., but 0 found

File: somefile.py (hunk 4 above)
Comment (stickler-ci-dev) [inline somefile.py]: E901 SyntaxError: invalid syntax

**Comment by stickler-ci-dev** [comment]
The following commits had issues. The pattern (?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

**Comment by stickler-ci-dev** [comment]
The following commits had issues. The pattern (?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

**Comment by stickler-ci-dev** [comment]
The following commits had issues. The pattern (?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

**Comment by stickler-ci-dev** [comment]
The following commits had issues. The pattern (?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

**Comment by stickler-ci-dev** [comment]
The following commits had issues. The pattern (?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

**Comment by stickler-ci-dev** [comment]
The following commits had issues. The pattern (?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

File: somefile.py (hunk 1 above)
Comment (stickler-ci-dev) [inline somefile.py]: E302 expected 2 blank lines, found 1

File: somefile.py (hunk 2 above)
Comment (stickler-ci-dev) [inline somefile.py]: E111 indentation is not a multiple of four

File: somefile.py (hunk 3 above)
Comment (stickler-ci-dev) [inline somefile.py]: E302 expected 2 blank lines, found 1

Comment (stickler-ci-dev) [inline somefile.py]: E203 whitespace before ':'

File: somefile.py (hunk 4 above)
Comment (stickler-ci-dev) [inline somefile.py]: E128 continuation line under-indented for visual indent

**Comment by stickler-ci-dev** [comment]
The following commits had issues. The pattern (?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

File: bad.php (hunk 5 above)
Comment (stickler-ci-dev) [inline bad.php]: Missing function doc comment

Comment (stickler-ci-dev) [inline bad.php]: Function name "bad_braces" is not in camel caps format

Comment (stickler-ci-dev) [inline bad.php]: Function name "bad_braces" is prefixed with a package name but does not begin with a capital letter

Comment (stickler-ci-dev) [inline bad.php]: Opening brace should be on a new line

File: bad.php (hunk 6 above)
Comment (stickler-ci-dev) [inline bad.php]: Expected 1 space before ., but 0 found

Comment (stickler-ci-dev) [inline bad.php]: Expected 1 space after ., but 0 found

File: somefile.py (hunk 4 above)
Comment (stickler-ci-dev) [inline somefile.py]: E901 SyntaxError: invalid syntax

File: somefile.py (hunk 3 above)
Comment (markstory) [inline somefile.py]: E302 expected 2 blank lines, found 1
E203 whitespace before ':'

File: somefile.py (hunk 1 above)
Comment (markstory) [inline somefile.py]: E302 expected 2 blank lines, found 1

File: somefile.py (hunk 2 above)
Comment (markstory) [inline somefile.py]: E111 indentation is not a multiple of four

File: somefile.py (hunk 4 above)
Comment (markstory) [inline somefile.py]: E128 continuation line under-indented for visual indent

**Comment by markstory** [comment]
The following commits had issues. The pattern '(?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+)' was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

**Comment by markstory** [comment]
The following commits had issues. The pattern '(?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+)' was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

**Comment by markstory** [comment]
The following commits had issues. The pattern '(?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+)' was not found in:
* 7accfe18f106e0dac1c67e10abaf5ef63455fbf3
* 222ea54d485402db557ae1e06f5b5a8ebde76957

**Pull request closed without merging**